### ISSUE TITLE
Voxelmorph with noisy image and noise-free image

### ISSUE THREAD

**Issue by s8908271** [voxelmorph/voxelmorph#343]
I have some problem as my topic, i try to use voxelmorph to register noisy image and noise-free image, it usually gets some wired registration, the intensity changes a lots.
fixed image:
https://i.imgur.com/2uFb3Ef.png
moving image:
https://i.imgur.com/AdyQdPh.png
moved image:
https://i.imgur.com/1Alf5pk.jpg

the parameter I used:

vol_shape = (256,256,96)
nb_features = [
    [16, 32, 32, 32],
    [32, 32, 32, 32, 32, 16, 16]
]

losses = [dice_coef_loss, vxm.losses.Grad('l2').loss]
loss_weights = [1, 0.01]
vxm_model = vxm.networks.VxmDense(vol_shape, nb_features, int_steps=0)
vxm_model.compile(optimizer=tf.keras.optimizers.Adam(lr=1e-4), loss=losses, loss_weights=loss_weights)

training only for 50 epochs

sorry for bad English

**Comment by adalca** [comment]
@s8908271 what is `dice_coef_loss` in your code? I would expect that you could use something like NCC and it might do ok.
